diff --git a/src/components/bottomSheetModal/BottomSheetModal.tsx b/src/components/bottomSheetModal/BottomSheetModal.tsx
index 5d131f1ed227fefa637dd655b5d0e41a328aa8d5..5831f5857c67b520293e798385ff0810959d569b 100644
--- a/src/components/bottomSheetModal/BottomSheetModal.tsx
+++ b/src/components/bottomSheetModal/BottomSheetModal.tsx
@@ -475,6 +475,7 @@ const BottomSheetModal = memo(forwardRef(BottomSheetModalComponent)) as <
 ) => ReturnType<typeof BottomSheetModalComponent>;
 (
   BottomSheetModal as React.MemoExoticComponent<
+    // @ts-ignore
     typeof BottomSheetModalComponent
   >
 ).displayName = 'BottomSheetModal';
diff --git a/src/constants.ts b/src/constants.ts
index 41591325fd7828b1184f5fd6d793ae0e36f224c2..e23c9f57d42b3e4f2292d28bae1269d91e5a3aa5 100644
--- a/src/constants.ts
+++ b/src/constants.ts
@@ -69,6 +69,7 @@ enum SNAP_POINT_TYPE {
   DYNAMIC = 1,
 }
 
+// @ts-ignore
 const ANIMATION_EASING: Animated.EasingFunction = Easing.out(Easing.exp);
 const ANIMATION_DURATION = 250;
 
diff --git a/src/hooks/useStableCallback.ts b/src/hooks/useStableCallback.ts
index 9202b4bf48bef2d1a03b9b279f9fb6a228560421..1be685f62644b7d0c7b6b0a77ed9b33327c970f0 100644
--- a/src/hooks/useStableCallback.ts
+++ b/src/hooks/useStableCallback.ts
@@ -8,6 +8,7 @@ type Callback<T extends unknown[], R> = (...args: T) => R;
 export function useStableCallback<T extends unknown[], R>(
   callback: Callback<T, R>
 ) {
+    // @ts-ignore
   const callbackRef = useRef<Callback<T, R>>();
 
   useLayoutEffect(() => {
